Stripe/STPPaymentContext.m

```diff
@@ -347,10 +497,51 @@ - (PKPaymentRequest *)buildPaymentRequest {
     NSArray<PKPaymentSummaryItem *> *summaryItems = self.paymentSummaryItems;
     paymentRequest.paymentSummaryItems = summaryItems;
     paymentRequest.requiredBillingAddressFields = [STPAddress applePayAddressFieldsFromBillingAddressFields:self.configuration.requiredBillingAddressFields];
+    paymentRequest.requiredShippingAddressFields = self.configuration.requiredShippingAddressFields;
     paymentRequest.currencyCode = self.paymentCurrency.uppercaseString;
+    if (self.selectedShippingMethod != nil) {
+        NSMutableArray<PKShippingMethod *>* orderedShippingMethods = [self.shippingMethods mutableCopy];
+        [orderedShippingMethods removeObject:self.selectedShippingMethod];
+        [orderedShippingMethods insertObject:self.selectedShippingMethod atIndex:0];
+        paymentRequest.shippingMethods = orderedShippingMethods;
+    }
+    else {
+        paymentRequest.shippingMethods = self.shippingMethods;
+    }
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_8_3
+    if ([paymentRequest respondsToSelector:@selector(shippingType)]) {
+        paymentRequest.shippingType = [[self class] pkShippingType:self.configuration.shippingType];;
+    }
+#endif
+    if (self.shippingAddress != nil) {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_9_0
+        if ([paymentRequest respondsToSelector:@selector(shippingContact)]) {
+            paymentRequest.shippingContact = [self.shippingAddress PKContactValue];
```

Comment:
Makes sense. Maybe add a comment saying this in the code itself with a link to the radar?

Also, I don't think we need to check for these `__IPHONE_OS_VERSION_MAX_ALLOWED` keys here though (and a few lines above), we aren't really supporting any Xcode's before version 7 and so everyone building this should have an sdk of at least 9 I believe? There are plenty of other places in the SDK where we assume your compiler knows about newer-sdk selectors.